Lib-DNDBeyond/dndb_transformSkill.command

```diff
@@ -1,5 +1,6 @@
 [h: toon = arg (0)]
 [h: skill = arg (1)]
+[h: skillMods = arg (2)]
```

Comment:
I'm pretty sure I've simplified the skill DTO enough to refactor this entire macro away.

Reply:
Fine the way it is, for now. Push this task down the road